cmd/loki/loki-local-config.yaml

```diff
@@ -4,22 +4,20 @@ server:
   http_listen_port: 3100
   grpc_listen_port: 9096
 
+common:
+  path_prefix: /tmp/loki
+  storage:
+    filesystem:
+      directory: /tmp/loki/store
+
 ingester:
-  wal:
-    enabled: true
-    dir: /tmp/wal
   lifecycler:
-    address: 127.0.0.1
     ring:
```

Comment:
same: WDYT of using a common/ring and removing this `ingester/lifecycler/ring section?: